stdlib/@tests/test_cases/check_re.py

```diff
@@ -2,18 +2,18 @@
 
 import mmap
 import re
-import typing as t
+from typing import AnyStr, Match
```

Comment:
I'm assuming that importing `Match` from `typing` rather than `re` here is intended for the test.